analysis/SConstruct

```diff
@@ -60,7 +62,10 @@ debrief_env = {'MAXIT' : 10,
                # Check if the repo requires LFS:
                'lfs_required' : gs.misc.load_yaml_value('config_global.yaml', 'prereq_git-lfs'),
                # .gitattributes location: 
-               'git_attrib_path' : '../.gitattributes'}
+               'git_attrib_path' : '../.gitattributes', 
+               # state of repo log file location:
+               'log' : 'state_of_repo.log'}
```

Comment:
A very minor issue: can we make the comments more consistent among no punctuation, `:` and `;`?